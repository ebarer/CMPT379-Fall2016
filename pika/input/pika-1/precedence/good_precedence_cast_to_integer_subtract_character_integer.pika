exec {

  print [ ^l^ | int ] - 87, _n_.
  print [ ^f^ | int ] - 42, _n_.
  print [ ^+^ | int ] - 55, _n_.
  print [ ^'^ | int ] - 4, _n_.
  print [ ^1^ | int ] - 49, _n_.
  print [ ^h^ | int ] - 19, _n_.
  print [ ^<^ | int ] - 28, _n_.
  print [ ^^^ | int ] - 17, _n_.
  print [ ^G^ | int ] - 60, _n_.
  print [ ^5^ | int ] - 25, _n_.
  print [ ^*^ | int ] - 90, _n_.
  print [ ^}^ | int ] - 89, _n_.

  print 54 - [ ^m^ | int ], _n_.
  print 82 - [ ^o^ | int ], _n_.
  print 6 - [ ^e^ | int ], _n_.
  print 53 - [ ^?^ | int ], _n_.
  print 39 - [ ^d^ | int ], _n_.
  print 8 - [ ^x^ | int ], _n_.
  print 87 - [ ^4^ | int ], _n_.
  print 12 - [ ^u^ | int ], _n_.
  print 75 - [ ^H^ | int ], _n_.
  print 33 - [ ^D^ | int ], _n_.
  print 97 - [ ^K^ | int ], _n_.
  print 88 - [ ^i^ | int ], _n_.

  print [ ^Y^ | int ] - [ ^(^ | int ], _n_.
  print [ ^"^ | int ] - [ ^M^ | int ], _n_.
  print [ ^b^ | int ] - [ ^H^ | int ], _n_.
  print [ ^@^ | int ] - [ ^`^ | int ], _n_.
  print [ ^_^ | int ] - [ ^`^ | int ], _n_.
  print [ ^1^ | int ] - [ ^a^ | int ], _n_.
  print [ ^A^ | int ] - [ ^g^ | int ], _n_.
  print [ ^k^ | int ] - [ ^+^ | int ], _n_.
  print [ ^g^ | int ] - [ ^)^ | int ], _n_.
  print [ ^i^ | int ] - [ ^.^ | int ], _n_.
  print [ ^?^ | int ] - [ ^n^ | int ], _n_.
  print [ ^U^ | int ] - [ ^?^ | int ], _n_.

}

