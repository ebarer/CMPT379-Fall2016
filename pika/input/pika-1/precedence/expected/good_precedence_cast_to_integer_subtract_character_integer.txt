21
60
-12
35
0
85
32
77
11
28
-48
36
-55
-29
-95
-10
-61
-112
35
-105
3
-35
22
-17
49
-43
26
-32
-1
-48
-38
64
62
59
-47
22
